datahub-web-react/src/graphql/chart.graphql

```diff
@@ -49,7 +49,7 @@ query getChart($urn: String!) {
         domain {
             ...entityDomain
         }
-        application {
+        applications {
```

Comment:
so, this constitutes a minor, but still real, breaking change. lets make sure we include this in the next version release notes